SU2_CFD/src/drivers/CDriver.cpp

```diff
@@ -32,12 +32,58 @@
+#include "../../include/solvers/CAdjNSSolver.hpp"
+#include "../../include/solvers/CAdjTurbSolver.hpp"
+#include "../../include/solvers/CHeatSolverFVM.hpp"
+#include "../../include/solvers/CFEASolver.hpp"
+#include "../../include/solvers/CTemplateSolver.hpp"
+#include "../../include/solvers/CDiscAdjSolver.hpp"
+#include "../../include/solvers/CDiscAdjFEASolver.hpp"
+#include "../../include/solvers/CFEM_DG_EulerSolver.hpp"
+#include "../../include/solvers/CFEM_DG_NSSolver.hpp"
+#include "../../include/solvers/CDiscAdjMeshSolver.hpp"
+#include "../../include/solvers/CMeshSolver.hpp"
+
+#include "../../include/interfaces/cfd/CConservativeVarsInterface.hpp"
+#include "../../include/interfaces/cfd/CMixingPlaneInterface.hpp"
+#include "../../include/interfaces/cfd/CSlidingInterface.hpp"
+#include "../../include/interfaces/cht/CConjugateHeatInterface.hpp"
+#include "../../include/interfaces/fsi/CDisplacementsInterface.hpp"
+#include "../../include/interfaces/fsi/CFlowTractionInterface.hpp"
+#include "../../include/interfaces/fsi/CDiscAdjFlowTractionInterface.hpp"
+#include "../../include/interfaces/fsi/CDisplacementsInterfaceLegacy.hpp"
+#include "../../include/interfaces/fsi/CDiscAdjDisplacementsInterfaceLegacy.hpp"
+
+#include "../../include/numerics/template.hpp"
+#include "../../include/numerics/transition.hpp"
+#include "../../include/numerics/heat.hpp"
+#include "../../include/numerics/flow/convection/roe.hpp"
+#include "../../include/numerics/flow/convection/fds.hpp"
+#include "../../include/numerics/flow/convection/fvs.hpp"
+#include "../../include/numerics/flow/convection/cusp.hpp"
+#include "../../include/numerics/flow/convection/hllc.hpp"
+#include "../../include/numerics/flow/convection/ausm_slau.hpp"
+#include "../../include/numerics/flow/convection/centered.hpp"
+#include "../../include/numerics/flow/flow_diffusion.hpp"
+#include "../../include/numerics/flow/flow_sources.hpp"
+#include "../../include/numerics/continuous_adjoint/adj_convection.hpp"
+#include "../../include/numerics/continuous_adjoint/adj_diffusion.hpp"
+#include "../../include/numerics/continuous_adjoint/adj_sources.hpp"
+#include "../../include/numerics/turbulent/turb_convection.hpp"
+#include "../../include/numerics/turbulent/turb_diffusion.hpp"
+#include "../../include/numerics/turbulent/turb_sources.hpp"
+#include "../../include/numerics/elasticity/CFEAElasticity.hpp"
 #include "../../include/numerics/elasticity/CFEALinearElasticity.hpp"
-#include "../../include/numerics/elasticity/CFEAMeshElasticity.hpp"
-#include "../../include/numerics/elasticity/CFEM_NeoHookean_Comp.hpp"
-#include "../../include/numerics/elasticity/CFEM_Knowles_NearInc.hpp"
-#include "../../include/numerics/elasticity/CFEM_IdealDE.hpp"
-#include "../../include/numerics/elasticity/CFEM_DielectricElastomer.hpp"
+#include "../../include/numerics/elasticity/CFEANonlinearElasticity.hpp"
+#include "../../include/numerics/elasticity/nonlinear_models.hpp"
```

Comment:
On the subject of restructuring the CNumerics files, I initially did it one-class-per-file with some sub folders, the problem with that is that we have some 85 of those classes... so 170 files which I think is maybe too many, more important is that some of those classes form a smaller hierarchies which are not clearly identifiable with so many files.
So I did it by "themes" instead, I think this division should inspire some re-usage when new classes are added, the average cpp size is under 1k lines, which I think is quite manageable.